web_src/js/components/ActivityHeatmap.vue

```diff
@@ -15,7 +15,7 @@
   </div>
 </template>
 <script>
```

Comment:
Let's take this opportunity to switch to `<script setup>` together

Reply:
Dropping https://vuejs.org/api/sfc-script-setup.html so I can come back to learn more about it once I can make time to continue working on this PR (summer holidays ahead over here).